exotic/api/ld.py

```diff
@@ -118,9 +116,15 @@ def check_standard(self, filter_: dict = None, loose: bool = False, loose_len: i
                 # match 'name' or 'filter' (strict) to actual name abbreviation, e.g. 'name'
                 elif filter_['name'] == f['name'].strip().upper() or \
                         (filter_['filter'] and filter_['filter'][:5].upper() == f['name'].strip().upper()):
+                    if filter_['name'] in LimbDarkening.filter_nonspecific_ids or \
+                            (filter_['filter'] and filter_['filter'][:5].upper() in LimbDarkening.filter_nonspecific_ids):
+                        if filter_['wl_min'] and filter_['wl_max'] and LimbDarkening.check_fwhm(filter_):
+                            filter_alias = f
+                            filter_alias['fwhm'] = (filter_['wl_min'], filter_['wl_max'])
+                            break
                     # exclude unknown vals for 'name'
-                    if filter_['name'] in filter_nonspecific_names or \
-                            (filter_['filter'] and filter_['filter'][:5].upper() in filter_nonspecific_names):
+                    elif filter_['name'] in LimbDarkening.filter_generic_ids or \
```

Comment:
Recommend reverting this stuff. The reason being is that AAVSO recommends ignoring the full list of `nonspecific` names. So that means it would only match MObs CV is either the desc said `MObs CV` or the FWHM vals match. So I would perform a check against those `nonspecific` values at the conclusion of the testing and then fill in the blanks for the description as necessary. ... We could even make a map to replace the description if it's empty.